Seneca-CDOT/telescope
# Clean up SSO code in web app, logout works

## Issue This PR Addresses

1. Fixes #674 
2. Fixes #695 
3. Fixes #694 
4. Fixes #646 

## Type of Change


## Description

Now that the SSO code is working with Seneca, I wanted to clean up a bunch of loose ends.  In the process I was also able to get Logout to work (hurray!).

We no longer need to generate dev certs, so I've removed that script from `tools/` and all callers of it.

To test this PR:

- run the login docker container locally (and redis, elastic if you need them via docker)
- build the frontend: `npm run build`
- run our node server: `npm start`
- browse to http://localhost:3000
- Click Login
- login as user1 with password user1pass
- You should go back to the homepage and be logged in
- Click Logout
- You should go back to the homepage and be logged out

I also moved the `/health` route to its own file to clean up our `app.js` and added GitHub commit info and git sha:

<img width="735" alt="Screen Shot 2020-03-26 at 2 27 45 PM" src="https://user-images.githubusercontent.com/427398/77693140-3e4cbb80-6f7e-11ea-8337-54134669d99f.png">

Hopefully this helps with what @c3ho has been fighting with in our build.

NOTE: I have not tested this against Seneca's SSO, because we have to merge to know if it works there.
